templates/openstack/consulting.html

```diff
@@ -1,389 +1,501 @@
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/baeb8320-ntt-logo.svg",
+            alt="",
+            width="130",
+            height="47",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "logo-section__logo"}
             ) | safe
           }}
-        </li>
-        <li class="p-inline-images__item">
-          {{
-            image(
-              url="https://assets.ubuntu.com/v1/d0b7aa28-logo-deutsche-telekom.png",
-              alt="Deutsche Telekom",
-              height="48",
-              width="96",
-              hi_def=True,
-              attrs={"class": "p-inline-images__logo"},
-              loading="lazy",
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/b41ec3f5-SBI-Bits_logo.png",
+            alt="",
+            width="615",
+            height="187",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "logo-section__logo"}
             ) | safe
           }}
-        </li>
-        <li class="p-inline-images__item">
-          {{
-            image(
-              url="https://assets.ubuntu.com/v1/5482b658-liveperson-logo.jpg",
-              alt="Liveperson",
-              height="33",
-              width="144",
-              hi_def=True,
-              attrs={"class": "p-inline-images__logo"},
-              loading="lazy",
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/5af38159-etisalat_logo.svg",
+            alt="",
+            width="108",
+            height="41",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "logo-section__logo"}
             ) | safe
           }}
-        </li>
-        <li class="p-inline-images__item">
-          {{
-            image(
-              url="https://assets.ubuntu.com/v1/4d6054f9-logo-cisco.svg",
-              alt="Cisco",
-              height="76",
-              width="144",
-              hi_def=True,
-              attrs={"class": "p-inline-images__logo"},
-              loading="lazy",
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/f7651dc9-Logo-GARR-300dpi.png",
+            alt="",
+            width="2417",
+            height="917",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "logo-section__logo"}
             ) | safe
           }}
-        </li>
-        <li class="p-inline-images__item">
-          {{
-            image(
-              url="https://assets.ubuntu.com/v1/b7693339-logo-bloomberg.svg",
-              alt="Bloomberg",
-              height="28",
-              width="144",
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
```

Comment:
```suggestion
<section class="p-strip u-no-padding--top">
```
To align padding around the logo section above